src/NUnitFramework/tests/nunit.framework.tests.csproj

```diff
@@ -1,52 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-<PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
-    <TargetFrameworks>net35;net40;net45;net46;netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
-    <TargetFrameworks>net35;net45;net46;netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
-  </PropertyGroup>
-
   <PropertyGroup>
     <RootNamespace>NUnit.Framework</RootNamespace>
+    <TargetFrameworks>net45;net46;netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
Feels like something I would place above `RootNamespace`, but I don't have a rationale other than a vague sense of the primacy of the `TargetFrameworks` element.

Reply:
Done